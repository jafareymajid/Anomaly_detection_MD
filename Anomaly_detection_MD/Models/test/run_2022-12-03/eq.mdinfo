
 NSTEP =     1000   TIME(PS) =       4.000  TEMP(K) =   287.48  PRESS = -2572.7
 Etot   =    -21440.5291  EKtot   =      2016.8811  EPtot      =    -23457.4102
 BOND   =        35.4903  ANGLE   =       158.8621  DIHED      =       167.9619
 1-4 NB =        57.7547  1-4 EEL =       985.7089  VDWAALS    =      6654.1328
 EELEC  =    -31547.6260  EHBOND  =         0.0000  RESTRAINT  =        30.3052
 EAMBER (non-restraint)  =    -23487.7154
 EKCMT  =       962.2866  VIRIAL  =      3349.2917  VOLUME     =     42972.5327
                                                    Density    =         0.8159
 Ewald error estimate:   0.3636E-03

| Final Performance Info:
| -----------------------------------------------------
| Average timings for last    1000 steps:
|     Elapsed(s) =      56.02 Per Step(ms) =      56.02
|         ns/day =       3.08   seconds/ns =   28008.89
|
| Average timings for all steps:
|     Elapsed(s) =      56.02 Per Step(ms) =      56.02
|         ns/day =       3.08   seconds/ns =   28008.89
| -----------------------------------------------------

