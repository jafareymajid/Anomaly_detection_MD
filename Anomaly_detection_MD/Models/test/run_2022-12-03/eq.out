
          -------------------------------------------------------
          Amber 18 SANDER                              2018
          -------------------------------------------------------

| Run on 12/03/2022 at 10:22:30

|   Executable path: sander
| Working directory: /mnt/e/MSU/Fall2022/CMSE802/project/scripts/test_software/run_2022-12-03
|          Hostname: Unknown
  [-O]verwriting output

File Assignments:
|  MDIN: eq.in                                                                 
| MDOUT: eq.out                                                                
|INPCRD: heat.rst7                                                             
|  PARM: protein.parm7                                                         
|RESTRT: eq.rst7                                                               
|  REFC: protein.rst7                                                          
| MDVEL: mdvel                                                                 
| MDFRC: mdfrc                                                                 
|  MDEN: mden                                                                  
| MDCRD: eq.nc                                                                 
|MDINFO: eq.mdinfo                                                             
|  MTMD: mtmd                                                                  
|INPDIP: inpdip                                                                
|RSTDIP: rstdip                                                                
|INPTRA: inptraj                                                               


 Here is the input file:

constant P                                                                     
 &cntrl                                                                        
  imin=0, ntx=5, irest=1,                                                      
  ntb=2, ntc=2, ntf=2,                                                         
  ntp=1, pres0=1.01325, taup=5,                                                
  nstlim=1000, dt=0.002, cut=9.0,                                              
  ntt=3, temp0=300, gamma_ln=1,                                                
  ntpr=1000, ntwx=10000, ntwr=10000, tempi=300,ioutfm=1,ig=-1,                 
  ntr=1,                                                                       
  restraint_wt=10.0,              ! Restraint force kcal/mol                   
  restraintmask="(:1-13)&(@CA,N,O)",   !put restraint on protein backbone      
 /                                                                             
 &ewald                                                                        
 /                                                                             
                                                                               

--------------------------------------------------------------------------------
   1.  RESOURCE   USE: 
--------------------------------------------------------------------------------

| Flags:                                                                        
 getting box info from netcdf restart file
| NetCDF restart box info found
|Largest sphere to fit in unit cell has radius =    15.883
| New format PARM file being parsed.
| Version =    1.000 Date = 12/03/22 Time = 10:21:12
 NATOM  =    4536 NTYPES =      15 NBONH =    3349 MBONA  =    1174
 NTHETH =     351 MTHETA =     139 NPHIH =     593 MPHIA  =     436
 NHPARM =       0 NPARM  =       0 NNB   =    8942 NRES   =    1091
 NBONA  =    1174 NTHETA =     139 NPHIA =     436 NUMBND =      29
 NUMANG =      58 NPTRA  =      71 NATYP =      21 NPHB   =       1
 IFBOX  =       2 NMXRS  =      23 IFCAP =       0 NEXTRA =    1066
 NCOPY  =       0


|     Memory Use     Allocated
|     Real              410810
|     Hollerith          14701
|     Integer           192180
|     Max Pairs        2414966
|     nblistReal         54432
|     nblist Int        158027
|       Total            14493 kbytes

| Note: 1-4 EEL scale factors are being read from the topology file.

| Note: 1-4 VDW scale factors are being read from the topology file.
| Duplicated    0 dihedrals
| Duplicated    0 dihedrals

     BOX TYPE: TRUNCATED OCTAHEDRON
Note: ig = -1. Setting random seed to   397665 based on wallclock time in microseconds.

--------------------------------------------------------------------------------
   2.  CONTROL  DATA  FOR  THE  RUN
--------------------------------------------------------------------------------

default_name                                                                    

General flags:
     imin    =       0, nmropt  =       0

Nature and format of input:
     ntx     =       5, irest   =       1, ntrx    =       1

Nature and format of output:
     ntxo    =       2, ntpr    =    1000, ntrx    =       1, ntwr    =   10000
     iwrap   =       0, ntwx    =   10000, ntwv    =       0, ntwe    =       0
     ioutfm  =       1, ntwprt  =       0, idecomp =       0, rbornstat=      0

Potential function:
     ntf     =       2, ntb     =       2, igb     =       0, nsnb    =      25
     ipol    =       0, gbsa    =       0, iesp    =       0
     dielc   =   1.00000, cut     =   9.00000, intdiel =   1.00000

Frozen or restrained atoms:
     ibelly  =       0, ntr     =       1
     restraint_wt =  10.00000

Molecular dynamics:
     nstlim  =      1000, nscm    =         0, nrespa  =         1
     t       =   0.00000, dt      =   0.00200, vlimit  =  20.00000

Langevin dynamics temperature regulation:
     ig      =  397665
     temp0   = 300.00000, tempi   = 300.00000, gamma_ln=   1.00000

Pressure regulation:
     ntp     =       1
     pres0   =   1.01325, comp    =  44.60000, taup    =   5.00000

SHAKE:
     ntc     =       2, jfastw  =       0
     tol     =   0.00001

Extra-points options:
     frameon =       1, chngmask=       1

Ewald parameters:
     verbose =       0, ew_type =       0, nbflag  =       1, use_pme =       1
     vdwmeth =       1, eedmeth =       1, netfrc  =       1
     Box X =   38.905   Box Y =   38.905   Box Z =   38.905
     Alpha =  109.471   Beta  =  109.471   Gamma =  109.471
     NFFT1 =   40       NFFT2 =   40       NFFT3 =   40
     Cutoff=    9.000   Tol   =0.100E-04
     Ewald Coefficient =  0.30768
     Interpolation order =    4
|  INFO: Old style inpcrd file read


    LOADING THE CONSTRAINED ATOMS AS GROUPS


   5.  REFERENCE ATOM COORDINATES

  defa
     Mask (:1-13)&(@CA,N,O); matches    39 atoms
|      EXTRA_PTS, trim_bonds: num bonds BEFORE trim =  3349     0
|      EXTRA_PTS, trim_bonds: num bonds AFTER  trim =  3349     0
|      EXTRA_PTS, trim_bonds: num bonds BEFORE trim =  1174     0
|      EXTRA_PTS, trim_bonds: num bonds AFTER  trim =   108     0
|      EXTRA_PTS, trim_theta: num angle BEFORE trim =   351     0
|      EXTRA_PTS, trim_theta: num angle AFTER  trim =   351     0
|      EXTRA_PTS, trim_theta: num angle BEFORE trim =   139     0
|      EXTRA_PTS, trim_theta: num angle AFTER  trim =   139     0
|      EXTRA_PTS, trim_phi:  num diheds BEFORE trim =   593     0
|      EXTRA_PTS, trim_phi:  num diheds AFTER  trim =   593     0
|      EXTRA_PTS, trim_phi:  num diheds BEFORE trim =   436     0
|      EXTRA_PTS, trim_phi:  num diheds AFTER  trim =   436     0

--------------------------------------------------------------------------------
   3.  ATOMIC COORDINATES AND VELOCITIES
--------------------------------------------------------------------------------

default_name                                                                    
 begin time read from input coords =     2.000 ps

 Number of triangulated 3-point waters found:     1066

     Sum of charges from parm topology file =   6.00000583
     Assuming uniform neutralizing plasma

--------------------------------------------------------------------------------
   4.  RESULTS
--------------------------------------------------------------------------------

|  # of SOLUTE  degrees of freedom (RNDFP):   10259.
|  # of SOLVENT degrees of freedom (RNDFS):       0.
|  NDFMIN =   10259.     NUM_NOSHAKE =      0     CORRECTED RNDFP =   10259.
|  TOTAL # of degrees of freedom (RNDF) =   10259.
 ---------------------------------------------------
 APPROXIMATING switch and d/dx switch using CUBIC SPLINE INTERPOLATION
 using   5000.0 points per unit in tabled values
 TESTING RELATIVE ERROR over r ranging from 0.0 to cutoff
| CHECK switch(x): max rel err =   0.2738E-14   at   2.422500
| CHECK d/dx switch(x): max rel err =   0.8314E-11   at   2.736960
 ---------------------------------------------------
| Local SIZE OF NONBOND LIST =    1287201
| TOTAL SIZE OF NONBOND LIST =    1287201

 NSTEP =     1000   TIME(PS) =       4.000  TEMP(K) =   287.48  PRESS = -2572.7
 Etot   =    -21440.5291  EKtot   =      2016.8811  EPtot      =    -23457.4102
 BOND   =        35.4903  ANGLE   =       158.8621  DIHED      =       167.9619
 1-4 NB =        57.7547  1-4 EEL =       985.7089  VDWAALS    =      6654.1328
 EELEC  =    -31547.6260  EHBOND  =         0.0000  RESTRAINT  =        30.3052
 EAMBER (non-restraint)  =    -23487.7154
 EKCMT  =       962.2866  VIRIAL  =      3349.2917  VOLUME     =     42972.5327
                                                    Density    =         0.8159
 Ewald error estimate:   0.3636E-03
 ------------------------------------------------------------------------------


      A V E R A G E S   O V E R    1000 S T E P S


 NSTEP =     1000   TIME(PS) =       4.000  TEMP(K) =   243.49  PRESS = -2997.5
 Etot   =    -21961.9460  EKtot   =      1708.3050  EPtot      =    -23670.2510
 BOND   =        46.1957  ANGLE   =       169.0859  DIHED      =       182.2513
 1-4 NB =        58.9588  1-4 EEL =       981.0241  VDWAALS    =      6803.2172
 EELEC  =    -32027.2556  EHBOND  =         0.0000  RESTRAINT  =       116.2717
 EAMBER (non-restraint)  =    -23786.5227
 EKCMT  =       800.2046  VIRIAL  =      3657.0876  VOLUME     =     44094.4166
                                                    Density    =         0.7954
 Ewald error estimate:   0.5528E-03
 ------------------------------------------------------------------------------


      R M S  F L U C T U A T I O N S


 NSTEP =     1000   TIME(PS) =       4.000  TEMP(K) =    39.74  PRESS =   450.4
 Etot   =       416.4466  EKtot   =       278.8033  EPtot      =       204.2123
 BOND   =         9.2044  ANGLE   =        24.4202  DIHED      =        12.5149
 1-4 NB =         6.6528  1-4 EEL =        11.6593  VDWAALS    =       174.3522
 EELEC  =       478.8685  EHBOND  =         0.0000  RESTRAINT  =       222.7220
 EAMBER (non-restraint)  =       -18.5096
 EKCMT  =       148.7923  VIRIAL  =       375.0833  VOLUME     =       686.3904
                                                    Density    =         0.0124
 Ewald error estimate:   0.1453E-03
 ------------------------------------------------------------------------------


--------------------------------------------------------------------------------
   5.  TIMINGS
--------------------------------------------------------------------------------

|                Build the list             5.77 (99.49% of List )
|                Other                      0.03 ( 0.51% of List )
|             List time                  5.80 (10.67% of Nonbo)
|                   Short_ene time            40.24 (99.44% of Direc)
|                   Other                      0.23 ( 0.56% of Direc)
|                Direct Ewald time         40.46 (83.40% of Ewald)
|                Adjust Ewald time          0.34 ( 0.71% of Ewald)
|                Self Ewald time            0.00 ( 0.01% of Ewald)
|                   Fill Bspline coeffs        0.30 ( 4.04% of Recip)
|                   Fill charge grid           0.70 ( 9.50% of Recip)
|                   Scalar sum                 2.43 (33.15% of Recip)
|                   Grad sum                   1.35 (18.32% of Recip)
|                   FFT time                   2.57 (34.94% of Recip)
|                   Other                      0.00 ( 0.05% of Recip)
|                Recip Ewald time           7.34 (15.14% of Ewald)
|                Force Adjust               0.01 ( 0.02% of Ewald)
|                Virial junk                0.03 ( 0.06% of Ewald)
|                Other                      0.33 ( 0.67% of Ewald)
|             Ewald time                48.52 (89.32% of Nonbo)
|             Other                      0.00 ( 0.01% of Nonbo)
|          Nonbond force             54.32 (99.54% of Force)
|          Bond/Angle/Dihedral        0.24 ( 0.43% of Force)
|          Other                      0.01 ( 0.02% of Force)
|       Force time                54.57 (97.42% of Runmd)
|       Shake time                 0.23 ( 0.41% of Runmd)
|       Verlet update time         1.15 ( 2.05% of Runmd)
|       Ekcmr time                 0.03 ( 0.04% of Runmd)
|       Other                      0.05 ( 0.09% of Runmd)
|    Runmd Time                56.02 (99.90% of Total)
|    Other                      0.06 ( 0.10% of Total)
| Total time                56.07 (100.0% of ALL  )

| Number of list builds   :         87

| Highest rstack allocated:     268814
| Highest istack allocated:       4536

| Final Performance Info:
| -----------------------------------------------------
| Average timings for all steps:
|     Elapsed(s) =      56.02 Per Step(ms) =      56.02
|         ns/day =       3.08   seconds/ns =   28012.20
| -----------------------------------------------------

|           Job began  at 10:22:30.360  on 12/03/2022
|           Setup done at 10:22:30.417  on 12/03/2022
|           Run   done at 10:23:26.435  on 12/03/2022
|     wallclock() was called   46188 times
